### ISSUE TITLE
Support git-bisect in the test-suite in a friendly way

### ISSUE THREAD

**Issue by dmnks** [rpm-software-management/rpm#3097]
We could perhaps have a user-friendly way to do git-bisect with an ad-hoc test script that's not checked in to the repo, as suggested by @ffesti today.

It seems like `make atshell` (or directly calling `./mktree atshell <script.sh>`) could be used for this. I'll need to give it a thought or two on one of these "hack on your favorite project" Fridays but it shouldn't be terribly difficult.

**Comment by dmnks** [comment]
@pmatilai also suggested that we could have a slot in `rpmlocal.at` reserved for this that's never checked in.

**Comment by pmatilai** [comment]
Yeah I'd prefer a place for ad-hoc tests, because whenever you need to bisect something you almost certainly want to add it to the test-suite afterwards. So if it's already in the test format, you just move it around without having to rewrite from some custom logic.
